hypervisor/arch/x86/Kconfig

```diff
@@ -252,7 +252,12 @@ config ACPI_PARSE_ENABLED
 	bool "Enable ACPI runtime parsing"
 	default y
 	help
-	  Whether to parse the ACPI tables at runtime.
+	  ACRN hypervisor stores static assigned platform ACPI data for system
+	  usage. It is supposed to be generated by acrn-config tool for specific
+	  board, otherwise a generic platform ACPI data will be used. When set,
+	  hypervisor could parse platform ACPI at runtime to make sure of getting
+	  correct ACPI data. This config is not necessarily needed when build
+	  hypervisor with configuration XMLs.
```

Comment:
Platform specific information may be retrieved thru boot-time parse of ACPI table, or thru parse of off-line tool. This option enables the boot-time parse of ACPI table, and overwrite the information from off-line tool.